capstone-project/src/main/webapp/map/map.component.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * @fileoverview Define the map component of the map module.
+ */
+'use strict';
+
+angular.module('map').component('mapComponent', {
+    templateUrl: 'map/map.template.html',
+    controller: function($scope) {
+
+        // Define some hard-coded markers to be shown on the map
+        $scope.markers = [{
+            city: 'India',
+            lat: 23.200000,
+            long: 79.225487
+        }, {
+            city: 'Gorakhpur',
+            lat: 26.7588,
+            long: 83.3697
+        }];
+
+        let googleMapOption = {
+            zoom: 4,
+            center: new google.maps.LatLng(25, 80)
+        };
+
+        $scope.gMap = new google.maps.Map(document.getElementById('map-container'), googleMapOption);
+
+        // Add a marker to the map
+        let createMarker = function(marker) {
```

Comment:
1. Can this be const instead of let? 

2. Naming is confusing 'createMarker' suggest the function returns a Marker. However, the function adds a marker to the map (as suggested from the comment)